frontend-react/src/hooks/UseLookupTable.ts

```
@@ -63,28 +53,21 @@ export async function getLatestData<T>(
 ): Promise<T | T[]> {
     const endpointHeader = lookupTableApi.getTableData<T>(version, tableName);
 
-    try {
-        return await axios
-            .get<T>(endpointHeader.url, endpointHeader)
-            .then((response) => response.data);
-    } catch (e: any) {
-        console.trace(e);
-        showError(e.toString());
-        return [];
```

change here is to throw on error rather than returning []